ktor-client/ktor-client-features/ktor-client-json/ktor-client-json-tests/jvm/src/io/ktor/client/features/json/tests/JsonTest.kt

```diff
@@ -113,7 +113,7 @@ abstract class JsonTest : TestWithKtor() {
     @Test
     fun testCustomContentTypes() = testWithEngine(CIO) {
         configCustomContentTypeClient {
-            acceptContentTypes = listOf(customContentType)
+            accept(customContentType)
```

Comment:
Only the custom content type was accepted before the change, now we accept both custom and json

Reply:
Fixed.